ui/src/pages/Lore/Lore.tsx

```diff
@@ -31,32 +32,41 @@ export function Lore() {
     };
     fetchData();
   }, []);
-  console.log(lore[currentLoreSelection])
+
+  const isPlainText = () => {
+    setPlainText(!plainText);
+  }
+
+  let text = (depth: string) => plainText ? 'lore__no_effect' : 'lore__container' + [depth];
```

Comment:
call this var: textDepth

Reply:
my bad didnt mean the param, i meant the entire variable, `let text` should be `let textClassName = ...`

Reply:
lolol okkk ill change that rq

Reply:
It seems by having `lore__no_effect` be an empty classname, we can put whatever inside this string. Its up to you whether we should make it into `''` or something like `no__classname`. No point in trying to do something like this when we dont have an actual used classname with attributes for it

Reply:
I left it as undefined

Reply:
nice